conf/test/digiroad2.properties

```diff
@@ -7,9 +7,9 @@ digiroad2.useVVHGeometry=true
 digiroad2.VVHServiceHost=localhost:6080
 digiroad2.VVHRestApiEndPoint=http://localhost:6080/arcgis/rest/services/VVH_OTH_TEST/
 digiroad2.tierekisteriRestApiEndPoint=http://localhost:8080/api/tierekisteri/
+digiroad2.tierekisteriViiteRestApiEndPoint=http://172.17.204.33:8013/trrest/
```

Comment:
Avoid using references to internal IPs, use localhost and test reference API instead